src/main/java/edu/harvard/iq/dataverse/authorization/providers/oauth2/AbstractOAuth2AuthenticationProvider.java

```diff
@@ -120,13 +120,13 @@ public OAuth2UserRecord getUserRecord(String code, String state, String redirect
         final Response response = request.send();
         int responseCode = response.getCode();
         final String body = response.getBody();        
-        logger.fine("In getUserRecord. Body: " + body);
+        logger.log(Level.FINE, "In getUserRecord. Body: {0}", body);
```

Comment:
Why is this being changed from our standard way of logging, @michbarsinai ? If you want the style to change you first need to fight for it at http://guides.dataverse.org/en/4.8.2/developers/coding-style.html 😄 . Please create a new issue if you feel strongly about this.

Reply:
This type of logging is wasteful: the JVM concatenates the string, passes it to the logger, only to have the logger ignore it. Passing the message format and the argument array is much more efficient (It's an auto-fix in NetBeans, BTW).

Reply:
@michbarsinai again, if you feel strongly about it. Please open a GitHub issue. Otherwise, we can simply keep changing each other's styles back and forth. 😄

Reply:
Oh! You do feel strongly about this. Thanks for opening #4284.